### DISCUSSION TITLE
Is there a way to set default quality?

### DISCUSSION THREAD

**Discussion by Jtav1** [mediacms-io/mediacms#503] [Q&A]
Hi - just got this working, have a quick question. Is there a way I can set the default quality I want videos to load in when I play one? Whether its per account, or device, or server, whatever. It seems like most times I click a video on my mediacms instance picks a lower res to play even if a higher one is already encoded and I can pick it manually.

Thanks! Great software by the way, it works and looks fantastic.

**Comment by mgogoulos** [comment]
Hi, in case of adaptive streaming/HLS (if the backend API response with the video metadata has an HLS file) the frontend player will negotiate the best quality based on the connection - so if you are on a good network it should start with the best connection. If you are on a bad connection it will start with a low quality but should increase it if the quality gets increased as well

If HLS is not found (typical case would be that the installation didn't finish well and it didn't find Bento4 software which is breaking the video files into HLS files) it will start streaming the file through the last used quality (because there is no auto in this case). So if you have seen a video with 720p and you are watching another one, if the latter has 720p it will start with this.  There is some logic there to look if it finds that specific video quality on the next video so it should be possible to add a check there (start from best quality and look downwards) but it is not something that is configurable now!
